theano/tensor/nlinalg.py

```diff
@@ -625,9 +625,12 @@ def perform(self, node, inputs, outputs):
         (x,) = inputs
         (w, u, v) = outputs
         assert x.ndim == 2, "The input of svd function should be a matrix."
-        w[0], u[0], v[0] = self._numop(x,
-                                       self.full_matrices,
-                                       self.compute_uv)
+        if self.compute_uv:
+            w[0], u[0], v[0] = self._numop(x,
+                                           self.full_matrices,
+                                           self.compute_uv)
+        else:
+            u[0] = self._numop(x, self.full_matrices, self.compute_uv)
```

Comment:
I have numpy 1.12 and if `compute_uv = True` it returns only singular values. Which version of numpy does theano use for testing?